addon/utils/utils.js

```diff
@@ -22,3 +24,43 @@ export function updateSelectedItemsHash (selections, attrs) {
   }
   return _selections
 }
+
+export function normalizeSort (sort) {
+  if (!Ember.isPresent(sort)) {
+    return []
+  }
+  return sort.map(function (item) {
+    let key = item.value
+    let direction = item.direction === ':desc' ? '-' : ''
+    return `${direction}${key}`
+  })
+}
+
+export function listDefaultSort (items, sortProperties) {
+  if (!Ember.isPresent(sortProperties)) {
+    return
+  }
+
+  let normalizedSortProperties = sortProperties.map(sortProperty => {
+    let resultArray = []
+    if (sortProperty.startsWith('-')) {
+      resultArray.pushObject(sortProperty.slice(1))
+      resultArray.pushObject('desc')
+    } else {
+      resultArray.pushObject(sortProperty)
+      resultArray.pushObject('asc')
+    }
+    return resultArray
+  })
+
+  return Ember.A(items.slice().sort((itemA, itemB) => {
```

Comment:
Add a comment here explaining that slice is used to create a copy of the array so that the sort isn't applied to the original